src/tests/JIT/Directed/Convert/out_of_range_fp_to_int_conversions.cs

```diff
@@ -87,11 +87,11 @@ public static int ConvertDoubleToInt32(double x, FPtoIntegerConversionType t)
 
             switch (t)
             {
-                case FPtoIntegerConversionType.CONVERT_MANAGED_BACKWARD_COMPATIBLE_X86_X64:
                 case FPtoIntegerConversionType.CONVERT_BACKWARD_COMPATIBLE:
                 case FPtoIntegerConversionType.CONVERT_SENTINEL:
                     return (Double.IsNaN(x) || (x<int.MinValue) || (x > int.MaxValue)) ? int.MinValue: (int) x;
 
+                case FPtoIntegerConversionType.CONVERT_MANAGED_BACKWARD_COMPATIBLE_X86_X64:
```

Comment:
You should delete `CONVERT_MANAGED_BACKWARD_COMPATIBLE_X86_X64` from everywhere instead of this change. You are getting rid of the backward compatible x86/x64 behavior so it is not a thing after your change.

Reply:
True. I have removed it in my latest change.